### ISSUE TITLE
Selection of datasets

### ISSUE THREAD

**Issue by CodeRider26** [Go2Heart/EchoSight#21]
Hello, thank you very much for sharing the code, I would like to ask about what is in the graph of the iNaturalist 2021 data used in the paper experiment?

![Image](https://github.com/user-attachments/assets/9821a8bc-278a-4059-902d-64aa90b47ae9)

**Comment by Go2Heart** [comment]
You should check the Encyclopedic VQA's paper, in which they put:
> For iNat21, we sample our train images from their train set, and our val and test images from their validation set (their test set annotations are not publicly available)

**Comment by CodeRider26** [comment]
Thank you for your reply! I would also like to ask if the images used in the kb_index.faiss file in the creation knowledge_base are all available in the Encyclopedic-VQA image download link you provided, right?
